src/RMGGeneratorDecay0.cc

```diff
@@ -15,9 +15,15 @@
 
 #include "RMGGeneratorDecay0.hh"
 
+#include <limits>
+
+#include "Randomize.hh"
+
 #include "RMGConfig.hh"
 #include "RMGLog.hh"
 #include "RMGManager.hh"
+#include "RMGTools.hh"
+
 #if RMG_HAS_BXDECAY0
 #include "bxdecay0_g4/primary_generator_action.hh"
```

Comment:
```suggestion
#include "bxdecay0/bb_utils.h"
#include "bxdecay0_g4/primary_generator_action.hh"
```

(needed for a comment below)